stock_account_operating_unit/model/stock_quant.py

```diff
@@ -22,11 +22,11 @@ def _account_entry_move(self, move):
         if move.product_id.valuation == 'real_time':
             # Inter-operating unit moves do not accept to
             # from/to non-internal location
-            if (
+            if ((move.location_id.company_id or
+                 move.location_dest_id.company_id) and
```

Comment:
make it simple:

```python
              if (move.location_id.company_id and
              move.location_id.company_id ==
                    move.location_dest_id.company_id and
                    move.operating_unit_id != move.operating_unit_dest_id):
```

Reply:
Ok, It is fine, but it is as complicated as I did :smile: